ludwig/schema/metadata/configs/encoders.yaml

```diff
@@ -960,10 +960,9 @@ CTRL:
     type:
         short_description: Language model trained to condition on control codes that govern style, content and task-specific behavior.
         long_description:
-            The ctrl encoder loads a pretrained CTRL (default ctrl) model using the Hugging Face
-            transformers package. CTRL is a causal (unidirectional) transformer pre-trained using language
-            modeling on a very large corpus of ~140 GB of text data with the first token reserved as a
-            control code (such as Links, Books, Wikipedia etc.).
+            The `ctrl` encoder loads a pretrained [CTRL](https://arxiv.org/abs/1909.05858) (default `ctrl`) model using the Hugging
+            Face transformers package. CTRL is a conditional transformer language model trained to condition on control codes that
+            govern style, content, and task-specific behavior.
```

Comment:
I would bring back the `(such as Links, Books, Wikipedia etc.).` because I don't know what control codes are without that (assuming most Ludwig users are lazy like me and don't want to read the paper)